wallet/src/main/java/com/android/identity_credential/wallet/logging/Event.kt

```diff
@@ -0,0 +1,24 @@
+package com.android.identity_credential.wallet.logging
+
+import com.android.identity.cbor.Cbor
+import com.android.identity.cbor.annotation.CborSerializable
+import com.android.identity.util.Logger
+import kotlinx.datetime.Instant
+
+@CborSerializable
+sealed class Event(
+    open val timestamp: Instant,
+    open var eventId: Long = EventLogger.generateUniqueID(),
```

Comment:
This is used as a key in the storageEngine.

Reply:
I was thinking we'd use the timestamp as the primary key with some local per-instance counter appended but having a separate ID works too. I think it might be nicer to use an UUID / string (and call it `eventUuid` isntead) instead of a 64-bit integer but I don't think that's a very important detail. Up to you.

Reply:
Initially I was considering the possibility of a duplicate key if two events are being at the same time. This is a very low possibility so I removed eventId.